crates/sui-core/src/traffic_controller/nodefw_client.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021, Facebook, Inc. and its affiliates
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct BlockAddresses {
+    pub addresses: Vec<BlockAddress>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct BlockAddress {
+    pub source_address: String,
+    pub destination_port: u16,
+    pub ttl: u64,
+}
```

Comment:
These are exported (or can be) from my module already - would you consider using them directly?  It makes it a compile time safety if they ever change, at the expense of an import that links the two.  I can totally understand the desire to decouple it though, so if that is the desire, maybe these could be at least pub(crate) to prevent others from using it in error.

Reply:
Yeah it was my intention to import from your crate once your stuff is merged. Really don't wanna stack more PR's just to be able to import those structs, so will handle in a later PR